src/System.Linq/tests/SelectManyTests.cs

```diff
@@ -212,8 +212,8 @@ public void IndexCausingLastToBeSelected()
             Assert.Equal(source.Last().total, source.SelectMany((e, i) => i == 4 ? e.total : Enumerable.Empty<int?>()));
         }
 
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
```

Comment:
I gave bad guidance for these "too intensive" ones -- I think we do not need `[ActiveIssue]` as there is no bug. Just ` [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]`.
I will close #16750